crates/core/src/settings/mod.rs

```diff
@@ -311,6 +311,7 @@ pub struct HomeSettings {
 pub struct RefreshRateSettings {
     pub regular: u8,
     pub inverted: u8,
+    pub file_types: HashMap<String, RefreshRateSettings>,
```

Comment:
Would it be possible for `RefreshRateSettings` to be non-recursive?

We use the term *kind* instead of *type* (a reserved word) as you can see elsewhere in this file.

This new attribute shouldn't be serialized if empty.